### PR TITLE
Closes #150

### LINKED ISSUE #150: Consistent handling of vector-based approach in method and group columns

### PR DESCRIPTION
**Pull request by DavidBarke**
- no method argument was added to default methods
- restored color scheme for one method case (blue for prob, red for prob and conf)
- fixed match.arg in confint_betabinom.model_estimation
- estimate_cdf.default supports only one method instead of multiple methods. This is for consistency with the other default methods

### CODE REVIEW

File: R/plot_functions.R [hunk 1]
```diff
@@ -969,53 +972,65 @@ plot_mod.mixmod_em <- function(p_obj, x, title_trace = "Fit", ...) {
...
 #'
-#' mrr <- rank_regression(
-#'   tbl_john,
+#' rr <- rank_regression(
+#'   x = tbl_john$x,
+#'   y = tbl_john$prob,
+#'   status = tbl_john$status,
 #'   distribution = "weibull3",
-#'   conf_level = .90
+#'   conf_level = 0.9
```
Comment (Tim-TU) [inline R/plot_functions.R:995]: 0.90 ?

File: R/plot_functions.R [hunk 2]
```diff
@@ -969,53 +972,65 @@ plot_mod.mixmod_em <- function(p_obj, x, title_trace = "Fit", ...) {
...
 #' )
 #'
 #' plot_reg_weibull <- plot_mod(
 #'   p_obj = plot_weibull,
-#'   x = mrr,
+#'   x = tbl_john$x,
```
Comment (Tim-TU) [inline R/plot_functions.R:1000]: Line indent

File: R/plot_functions.R [hunk 3]
```diff
@@ -969,53 +972,65 @@ plot_mod.mixmod_em <- function(p_obj, x, title_trace = "Fit", ...) {
...
 #'
-#' mrr_ln <- rank_regression(
-#'   tbl_john,
+#' rr_ln <- rank_regression(
+#'   x = tbl_john$x,
+#'   y = tbl_john$prob,
+#'   status = tbl_john$status,
 #'   distribution = "lognormal3",
-#'   conf_level = .90
+#'   conf_level = 0.9
```
Comment (Tim-TU) [inline R/plot_functions.R:1026]: 0.90?

File: R/plot_functions.R [hunk 4]
```diff
@@ -969,53 +972,65 @@ plot_mod.mixmod_em <- function(p_obj, x, title_trace = "Fit", ...) {
...
 #' )
 #'
 #' plot_reg_lognormal <- plot_mod(
 #'   p_obj = plot_lognormal,
-#'   x = mrr_ln,
+#'   x = tbl_john$x,
+#'   loc_sc_params = rr_ln$loc_sc_params,
+#'   distribution = "lognormal3",
```
Comment (Tim-TU) [inline R/plot_functions.R:1033]: Line indent

**Review by Tim-TU — APPROVED** [review]

File: R/plot_functions.R (hunk 1 above)
Comment (DavidBarke) [reply R/plot_functions.R:995]: I thought we agreed on using leading 0 and removing trailing 0

File: R/plot_functions.R (hunk 2 above)
Comment (DavidBarke) [reply R/plot_functions.R:1000]: Could you please elaborate. I can't see any problem with the indentation.

File: R/plot_functions.R (hunk 4 above)
Comment (DavidBarke) [reply R/plot_functions.R:1033]: Same here

**Pull request merged**